pages/common/mv.md

```diff
@@ -33,4 +33,4 @@
 
 - Specify target directory (convenient in situations when the target directory has to be the first argument):
 
-`ls | {{parallel|xargs}} mv -t {{path/to/target_directory}}`
+`{{find /var/log -type f -name '*.log' -print0}} | {{xargs -0}} mv -t {{path/to/target_directory}}`
```

Comment:
```suggestion
`mv -t {{path/to/target_directory}} {{path/to/source1 path/to/source2 ...}}`
```

Maybe like that it would be more simple?

Reply:
It is more simple but it is like doing an easy job the hard way. Using `-t` is just redundant here as the destination can just be put at the end.

Anyway, I'll let the community decide/comment.